app-backend/api/src/main/scala/user/Routes.scala

```diff
@@ -78,7 +79,19 @@ trait UserRoutes extends Authentication with PaginationDirectives with UserError
     rejectEmptyResponse {
       complete {
         val authId = URLDecoder.decode(authIdEncoded, "US_ASCII")
-        Users.getUserWithOrgsById(authId)
+        Users.getUserWithOrgById(authId)
+      }
+    }
+  }
+
+  // TODO: Restrict to users with correct permissions, e.g. admin
+  def updateUserByEncodedAuthId(authIdEncoded: String): Route = authenticate { admin =>
+    entity(as[User.Update]) { updatedUser =>
+      onSuccess(Users.updateUser(updatedUser, authIdEncoded)) {
+        case 1 => complete(StatusCodes.NoContent)
+        case count => throw new IllegalStateException(
```

Comment:
Yeah I thought that might be the case, that was why I guessed it might not be germane to this PR. The only thing I can imagine that could cause this to be an issue would be a `pg_restore` over an existing database that ignores the schema, so probably fine to ignore